Newmusic

PROPHESSOR opened the pull request.

PROPHESSOR committed:
Первая попытка добавить разнообразия музыке

MorthimerMcMare committed:
Merge branch 'master' into newmusic

MorthimerMcMare committed:
Замена MIDI-музыки в INTRO и SEC2.

Обе -- из игры "Blackthorne" (так получилось. Пытался другие, не особо подошли).

PROPHESSOR commented on MapInfos/MapsInfo.txt:
```diff
@@ -58,42 +58,42 @@ Map SEC1 "Sector 1" {
 
 Map SEC2 "Sector 2" {
-    Music    = "D_DOOM"
+    Music    = "D_SEC2"
```
А где сам файл?

PROPHESSOR commented on MapInfos/MapsInfo.txt:
```diff
@@ -32,7 +32,7 @@ Map TITLEMAP3 "Introduction" {
 
 Map INTRO "Entrance" {
-    Music    = "D_DEAD"
+    Music    = "D_ENTRNC"
```
Аналогично

MorthimerMcMare committed:
Всё-таки добавил музыку в INTRO и SEC2.

PROPHESSOR commented:
Сорри, конечно, но эта музыка прям совсем не подходит... :confused:

MorthimerMcMare commented:
Только если звучит непривычно и немного выбивается по качеству в лучшую сторону. Однако граничных условий нет, субъективное видение у нас обоих никто не отменял...

PROPHESSOR commented:
@MorthimerMcMare ты продолжай, нужно как можно больше вариантов
Но именно эти не подходят, совсем меняют восприятие игры и не в лучшую сторону

MorthimerMcMare commented:
Хорошо, но объясни, почему ты так считаешь?

PROPHESSOR commented:
Как ты говоришь, субъективщина
Сложно объяснить, но треки D_DOOM и D_DEAD хорошо подходят, а эти - нет
Да и те, что я добавил, если честно, тоже, поэтому и лежат тут просто для вида

MorthimerMcMare commented:
А. А я как раз по твоим ориентировался.

Давай тестеров ещё спросим всё-таки.

PROPHESSOR commented:
А, прости, что дезинформировал, я пытался что-то подобрать, но так ничего и не нашел кроме того, что есть в мастере
А спрашивать нужно уже после того как у нас консенсус наступит, пока рановато, если я всё равно против

MorthimerMcMare committed:
Музыка на SEC3, SEC5 и SEC6.

Что-то точно из Resurgence, на SEC5 -- повторение из Blackthorne. Последняя не знаю, откуда.

PROPHESSOR committed:
Merge branch 'master' into newmusic

PROPHESSOR committed:
Merge branch 'master' into newmusic

PROPHESSOR committed:
Предложение по музыке от Loard Taurus

PROPHESSOR committed:
Отобрал наиболее звучащие версии на данный момент

PROPHESSOR committed:
E1M2 в SEC7

PROPHESSOR committed:
Merge branch 'master' into newmusic

PROPHESSOR committed:
Немного переделал E1M5 для REAC (промежуточная версия)
